debian/libgrpc0.lintian-overrides

```diff
@@ -0,0 +1 @@
+possible-gpl-code-linked-with-openssl
```

Comment:
I'm mildly interested in that one. What makes it think that there's GPL code anywhere near gRPC ? Do you know ?